lib/connection/statement.js

```diff
@@ -576,6 +576,15 @@ function BaseStatement(
     sendCancelStatement(context, statement, callback);
   };
 
+  //Integration Testing purpose.
```

Comment:
are we sure we need this function? can we test it in a different way to not add test-only purpose code in the production code?

Reply:
Actually, we cannot access the qcc located in sf.js at the result level. When we get the result, we should have qcc information on the result statement if we want to compare the qcc value after the query is completed.